nginx-controller/nginx/configurator.go

```diff
@@ -63,6 +67,48 @@ func (cnf *Configurator) addOrUpdateIngress(ingEx *IngressEx) {
 	nginxCfg := cnf.generateNginxCfg(ingEx, pems, jwtKeyFileName)
 	name := objectMetaToFileName(&ingEx.Ingress.ObjectMeta)
 	cnf.nginx.AddOrUpdateIngress(name, nginxCfg)
+
+	key := fmt.Sprintf(
```

Comment:
key generation happens in multiple places. better to create a function for it

Reply:
+